aws/table_aws_glue_catalog_table.go

```diff
@@ -0,0 +1,238 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/glue"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsGlueCatalogTable(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_glue_catalog_table",
+		Description: "AWS Glue Catalog Table",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "database_name"}),
+			ShouldIgnoreError: isNotFoundError([]string{"EntityNotFoundException"}),
+			Hydrate:           getGlueCatalogTable,
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listGlueCatalogDatabases,
+			Hydrate:       listGlueCatalogTables,
```

Comment:
Can we add optional quals for limiting the results?
In doc, I can see the `catalog_id`, `databse_name` etc can be use as optional quals.